src/Symfony/Component/Finder/Iterator/SortableIterator.php

```diff
@@ -27,6 +27,10 @@ class SortableIterator implements \IteratorAggregate
     public const SORT_BY_NAME_NATURAL = 6;
 
     private $iterator;
+
+    /**
+     * @var \Closure|null
```

Comment:
The existing type is `\Closure|int|null` (`\Closure|1|-1|null` to be precise with literal types)